<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Static4</class>
 <widget class="QWidget" name="Static4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gen 4 Static</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxProfile">
     <property name="title">
      <string>Profile</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="labelProfile">
        <property name="text">
         <string>Profile</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="labelProfileGame">
        <property name="text">
         <string>Game</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLabel" name="labelProfileGameValue"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="labelProfileTID">
        <property name="text">
         <string>TID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonProfileManager">
        <property name="text">
         <string>Manager</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelProfileTIDValue"/>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxProfiles"/>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="labelProfileSID">
        <property name="text">
         <string>SID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" rowspan="2">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="labelProfileSIDValue"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabRNGSelector">
     <widget class="QWidget" name="tabStatic">
      <attribute name="title">
       <string>Generator</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxGeneratorRNGInfo">
         <property name="title">
          <string>RNG Info</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="1">
           <widget class="ComboMenu" name="toolButtonGeneratorLead">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelGeneratorInitialAdvances">
            <property name="text">
             <string>Initial Advances</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="TextBox" name="textBoxGeneratorInitialAdvances">
            <property name="text">
             <string notr="true">0</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="TextBox" name="textBoxGeneratorMaxAdvances">
            <property name="text">
             <string notr="true">100000</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelGeneratorLead">
            <property name="text">
             <string>Lead</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelGeneratorMaxAdvances">
            <property name="text">
             <string>Max Advances</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0" colspan="2">
           <widget class="QPushButton" name="pushButtonGenerate">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Generate</string>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelGeneratorSeed">
            <property name="text">
             <string>Seed</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="TextBox" name="textBoxGeneratorSeed"/>
          </item>
          <item row="4" column="1">
           <widget class="TextBox" name="textBoxGeneratorDelay"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="labelGeneratorDelay">
            <property name="text">
             <string>Delay</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBoxGeneratorSettings">
         <property name="title">
          <string>Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <widget class="QLabel" name="labelGeneratorCategory">
            <property name="text">
             <string>Category</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="ComboBox" name="comboBoxGeneratorCategory">
            <item>
             <property name="text">
              <string>Starters</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Fossils</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Gifts</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Game Corner</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Stationary</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Legends</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Events</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Roamers</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelGeneratorPokemon">
            <property name="text">
             <string>Pokemon</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="ComboBox" name="comboBoxGeneratorPokemon"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelGeneratorLevel">
            <property name="text">
             <string>Level</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="spinBoxGeneratorLevel">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="groupBoxGeneratorFilters">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Filters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" rowspan="2">
           <widget class="Filter" name="filterGenerator" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="TableView" name="tableViewGenerator">
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSpreadSearcher">
      <attribute name="title">
       <string>Searcher</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_9">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBoxSearcherRNGInfo">
         <property name="title">
          <string>RNG Info</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="0" colspan="2">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <widget class="QLabel" name="labelSearcherMinDelay">
              <property name="text">
               <string>Min Delay</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="TextBox" name="textBoxSearcherMinDelay">
              <property name="text">
               <string notr="true">600</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="labelSearcherMaxDelay">
              <property name="text">
               <string>Max Delay</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <widget class="TextBox" name="textBoxSearcherMaxDelay">
              <property name="text">
               <string notr="true">2000</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelSearcherMinAdvance">
              <property name="text">
               <string>Min Advance</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="TextBox" name="textBoxSearcherMinAdvance">
              <property name="text">
               <string notr="true">0</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QLabel" name="labelSearcherMaxAdvance">
              <property name="text">
               <string>Max Advance</string>
              </property>
             </widget>
            </item>
            <item row="1" column="3">
             <widget class="TextBox" name="textBoxSearcherMaxAdvance">
              <property name="text">
               <string notr="true">1000</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0" colspan="2">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="pushButtonSearch">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="text">
               <string>Search</string>
              </property>
              <property name="checkable">
               <bool>false</bool>
              </property>
              <property name="autoDefault">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonCancel">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Cancel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="labelSearcherLead">
            <property name="text">
             <string>Lead</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="ComboBox" name="comboBoxSearcherLead">
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Cute Charm ♂</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Cute Charm ♀</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Synchronize</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QGroupBox" name="groupBoxSearcherFilters">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="title">
          <string>Filters</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0" rowspan="2">
           <widget class="Filter" name="filterSearcher" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBoxSearcherSettings">
         <property name="title">
          <string>Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="0" column="0">
           <widget class="QLabel" name="labelSearcherCategory">
            <property name="text">
             <string>Category</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelSearcherPokemon">
            <property name="text">
             <string>Pokemon</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="ComboBox" name="comboBoxSearcherPokemon"/>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="Line" name="line_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="labelSearcherLevel">
            <property name="text">
             <string>Level</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="ComboBox" name="comboBoxSearcherCategory">
            <item>
             <property name="text">
              <string>Starters</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Fossils</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Gifts</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Game Corner</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Stationary</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Legends</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Events</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QSpinBox" name="spinBoxSearcherLevel">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="3">
        <widget class="TableView" name="tableViewSearcher">
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextBox</class>
   <extends>QLineEdit</extends>
   <header>Form/Controls/TextBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>Form/Controls/TableView.hpp</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>Form/Controls/ComboBox.hpp</header>
  </customwidget>
  <customwidget>
   <class>Filter</class>
   <extends>QWidget</extends>
   <header>Form/Controls/Filter.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ComboMenu</class>
   <extends>QToolButton</extends>
   <header>Form/Controls/ComboMenu.hpp</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBoxProfiles</tabstop>
  <tabstop>pushButtonProfileManager</tabstop>
  <tabstop>tabRNGSelector</tabstop>
  <tabstop>toolButtonGeneratorLead</tabstop>
  <tabstop>textBoxGeneratorSeed</tabstop>
  <tabstop>textBoxGeneratorInitialAdvances</tabstop>
  <tabstop>textBoxGeneratorMaxAdvances</tabstop>
  <tabstop>textBoxGeneratorDelay</tabstop>
  <tabstop>pushButtonGenerate</tabstop>
  <tabstop>comboBoxGeneratorCategory</tabstop>
  <tabstop>comboBoxGeneratorPokemon</tabstop>
  <tabstop>spinBoxGeneratorLevel</tabstop>
  <tabstop>tableViewGenerator</tabstop>
  <tabstop>comboBoxSearcherLead</tabstop>
  <tabstop>textBoxSearcherMinDelay</tabstop>
  <tabstop>textBoxSearcherMaxDelay</tabstop>
  <tabstop>textBoxSearcherMinAdvance</tabstop>
  <tabstop>textBoxSearcherMaxAdvance</tabstop>
  <tabstop>pushButtonSearch</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>comboBoxSearcherCategory</tabstop>
  <tabstop>comboBoxSearcherPokemon</tabstop>
  <tabstop>spinBoxSearcherLevel</tabstop>
  <tabstop>tableViewSearcher</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
